### ISSUE TITLE
Add repository status to repository tree (feature request)

### ISSUE THREAD

**Issue by jirsbek** [ForkIssues/Tracker#379]
First of all, thank you for great app.

I would love to see in my repository tree the status of a repository. E.g. if there are some change to commit, current branch, count of not pushed commits etc.

I remember SourceTree has this feature and it's quite helpful.

![sourcetree2 0repobrowser](https://user-images.githubusercontent.com/8224496/43967601-09a7eaae-9cc5-11e8-9e46-3e3708767b21.png)

**Comment by tomi-engel** [comment]
I wanted to ask the same question (propose the same feature) addition.
I have clients with use a very fine grained repo layout and it is really hard to keep track of all the changes.

I do not know how to express support for such a feature (or donate for its implementation?) … so I just added my "thumb up" above.

**Comment by tuparev** [comment]
Yes, please, please, I am waiting for this already 2 years. This is the only feature that stops me switching  to Fork (because we have about 30-50 active repositories). I promise, not only I, but most of my colleagues will buy a licence!
